src/lib/chat/index.ts

```diff
@@ -131,24 +69,34 @@ export class Chat {
   }
 
   async setupConversations() {
-    const rooms = await this.client.getRoomsUserIsIn();
```

Comment:
The rest of the file is cleaning up the responsibilities between chat client, matrix adapter, and matrix client